bridgesync/downloader.go

```diff
@@ -102,6 +108,13 @@ func buildBridgeEventHandler(contract *polygonzkevmbridgev2.Polygonzkevmbridgev2
 			return fmt.Errorf("failed to extract bridge event calldata (tx hash: %s): %w", l.TxHash, err)
 		}
 
+		isNativeToken := false
```

Comment:
Nitpick, you could inline this and spare a couple of lines if you want.

```suggestion
		isNativeToken := bridgeEvent.OriginAddress == gasTokenAddress ||
			bridgeEvent.OriginAddress == common.Address(aggkitCommon.ZeroHash.Bytes())
```